tests/functional/spec/difference.spec.js

```diff
@@ -0,0 +1,73 @@
+import AgeBlockYearPage from "../generated_pages/difference_in_years/age-block.page";
```

Comment:
The spec name is not clear. I think we make a folder called placeholder and have placeholder related spec in there.

Reply:
Changed the spec names so they are more specific and descriptive, created `placeholder` folder in `features`, moved placeholder schemas specs in there.